index.html

```diff
@@ -14,13 +14,30 @@
             Adebsa
         </a>
         <nav>
```

Comment:
- [ ] Please reduce your Navigation `height` it looks bigger than the original design and affects your design please fix it in order to stick to the Figma template as much as possible :smile: 

#### Figma navigation
![Screenshot from 2022-09-29 21-43-41](https://user-images.githubusercontent.com/6615394/193148016-12af37a2-b74d-4d25-9446-0bd89d8a52b9.png)

#### Your design
![Screenshot from 2022-09-29 21-43-11](https://user-images.githubusercontent.com/6615394/193148021-90651c55-bdfc-4045-a8ef-4391bec7db1b.png)